readme.md

```diff
@@ -158,6 +158,28 @@ You can use the [Angular.js binding](https://github.com/hrajchert/angular-screen
 </div>
 ```
 
+#### Toggle Fullscren as an Angular 2 Directive
```

Comment:
I changed to Fullscreen the page with Angular 2 as my example is valid for a full page, not an element